tests/test_plugin.py

```diff
@@ -10,6 +10,13 @@
 from  delayed_assert import expect, assert_expectations
 
 from pytest_reportportal.plugin import pytest_configure
+from pytest_reportportal import RPLogger
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 4. You should put a blank line between each group of imports.


```
from  delayed_assert import assert_expectations, expect
import pytest

from pytest_reportportal import RPLogger
from pytest_reportportal.plugin import pytest_configure
```

Reply:
Hopefully I used right order)

Reply:
```
# These are imports from standard library
try:
    from unittest.mock import create_autospec, Mock, patch
except ImportError:
    from mock import create_autospec, Mock, patch

# These are third parties
from _pytest.config import Config
from  delayed_assert import assert_expectations, expect
import pytest
from requests.exceptions import RequestException

# These are application-specific
from pytest_reportportal.plugin import pytest_configure
from pytest_reportportal import RPLogger
```

Reply:
Better, but some imports are still not in alphabetical order.

Reply:
fixed